app_dart/lib/src/model/firestore/task.dart

```diff
@@ -245,3 +261,45 @@ class Task extends Document {
     return buf.toString();
   }
 }
+
+/// Generates task documents based on targets.
+List<Task> targetsToTaskDocuments(Commit commit, List<Target> targets) {
```

Comment:
nit: I think the name of the function is a little confusing and might be clearer as 'targetsToDocuments'.

Reply:
Are we converting targets or tasks?

Reply:
This is to convert targets to tasks